habitat_baselines/rl/requirements.txt

```diff
@@ -2,3 +2,6 @@ moviepy>=1.0.1
 torch>=1.3.1
 protobuf==3.20.1
 tensorboard==2.8.0
+
+faster_fifo==1.4.0
```

Comment:
Any reason these exact versions are pinned?

Reply:
Both are the most recent releases. This could probably be unpinned, but I think we've switched to pinning stuff for the CI.

Reply:
@erikwijmans Requirements.txt affects more than just CI though. It also affects local installs etc.

Reply:
@vincentpierre Do you have a preference here? IIRC you were advocating for pinning versions.

Reply:
@erikwijmans @vincentpierre One of my collaborators on pybind11 has a great writeup on why this is generally a bad idea for anything except CI testing: https://iscinumpy.dev/post/bound-version-constraints/

Reply:
I think CI versions should definitely be pinned. If we do not have a separate requirement.txt for CI, then I think we should pin it here. We can always increment the version later, but at least we will know that things should work with the versions we have.
Ideally we have pinned versions for CI and min versions (max if there is an issue with a version) in the requirements.

Reply:
@vincentpierre More worried about this causing issues for people installing it along with other packages. You can always put a more strict version pinning in the CI and it will keep pip satisifed. You can't be more lenient without editing the requirements.txt